lib/project_types/theme/commands/package.rb

```diff
@@ -53,6 +56,20 @@ def theme_name(path)
 
         [theme_name, theme_info["theme_version"]].compact.join("-")
       end
+
+      def command
+        return @command unless @command.nil?
+
+        @command = if @ctx.which(ZIP) == true
+          ZIP
+        elsif @ctx.which(SEVEN_ZIP) == true
+          SEVEN_ZIP
+        end
+      end
```

Comment:
The `which` method returns the path the command (e.g., @ctx.which(ZIP) returns `"/usr/bin/zip"` on my machine. So, we may change this method to something like this:

```suggestion
      def command
        return @command unless @command.nil?

        @command = if @ctx.which(ZIP)
          ZIP
        elsif @ctx.which(SEVEN_ZIP)
          SEVEN_ZIP
        end
      end
```